demo/admin/src/products/ProductForm.tsx

```diff
@@ -150,54 +179,119 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
-                        />
-                        <Field
-                            fullWidth
-                            name="tags"
-                            label="Tags"
-                            component={FinalFormSelect}
-                            multiple
-                            {...tagsSelectAsyncProps}
-                            getOptionLabel={(option: GQLProductTagsSelectFragment) => option.title}
-                        />
-                        <CheckboxField name="inStock" label={<FormattedMessage id="product.inStock" defaultMessage="In stock" />} fullWidth />
-                        <Field name="image" isEqual={isEqual}>
-                            {createFinalFormBlock(rootBlocks.image)}
-                        </Field>
-                    </MainContent>
-                </>
-            )}
+            {({ values, form }) => {
+                return (
+                    <>
+                        {saveConflict.dialogs}
+                        <MainContent>
+                            <TextField required fullWidth name="title" label={<FormattedMessage id="product.title" defaultMessage="Title" />} />
+                            <TextField required fullWidth name="slug" label={<FormattedMessage id="product.slug" defaultMessage="Slug" />} />
+                            <TextAreaField
+                                required
+                                fullWidth
+                                name="description"
+                                label={<FormattedMessage id="product.description" defaultMessage="Description" />}
+                            />
+                            <>
+                                <AsyncSelectField
+                                    name="manufacturerFilter"
+                                    loadOptions={async () => {
+                                        const { data } = await client.query<GQLManufacturerCountriesQuery, GQLManufacturerCountriesQueryVariables>({
+                                            query: gql`
+                                                query ManufacturerCountries {
+                                                    manufacturerCountries {
+                                                        nodes {
+                                                            id
+                                                            used
+                                                        }
+                                                    }
+                                                }
+                                            `,
+                                        });
+
+                                        return data.manufacturerCountries.nodes;
+                                    }}
+                                    getOptionLabel={(option: GQLManufacturerCountriesQuery["manufacturerCountries"]["nodes"][0]) => option.id}
```

Comment:
Isn't this the default implementation?

Reply:
no, sorry. If I followed the path correctly the default is an empty string: https://github.com/vivid-planet/comet/blob/next/packages/admin/admin/src/form/FinalFormSelect.tsx#L27

at least it matches the behaviour 
<img width="747" alt="image" src="https://github.com/vivid-planet/comet/assets/1324250/8f24358d-e9bd-41e4-988f-736cca01aadc">